service/awsconfig/v7/cloudformation/cloud_formation.go

```diff
@@ -0,0 +1,41 @@
+package cloudformation
+
+import (
+	"github.com/aws/aws-sdk-go/service/cloudformation"
+	"github.com/giantswarm/microerror"
+)
+
+type Config struct {
+	Client *cloudformation.CloudFormation
+}
+
+type CloudFormation struct {
+	client *cloudformation.CloudFormation
+}
+
+func New(config Config) (*CloudFormation, error) {
+	if config.Client == nil {
+		return nil, microerror.Maskf(invalidConfigError, "%T.Client must not be empty", config)
+	}
+
+	c := &CloudFormation{
+		client: config.Client,
+	}
+
+	return c, nil
+}
+
+// TODO
+func (c *CloudFormation) DescribeOutputs(stackName string) ([]*cloudformation.Output, error) {
```

Comment:
In order to drain nodes on AWS we need to access CF stack outputs in the lifecycle resource. The logic for this is already implemented in the cloudformation resource. I want to extract the functionality there and put it in a common place, which is the cloudformation service I added here. That way we can reuse the essential bits in multiple places. This PR though is only for preparation to have more nice PRs.